site/examples/progress_bars.js

```diff
@@ -5,6 +5,19 @@ const basic = `<div class='h12 bg-darken10 relative round-full'>
   <div class='absolute h12 bg-green-light round-full' style='width:50%;'></div>
 </div>`;
 
+const playback = `<div class='flex-parent flex-parent--row flex-parent--center-cross border border--gray-light round-t'>
+    <div class='flex-child py18 pr12 pl18 border-r border--gray-light bg-white cursor-pointer'>
+      <div class='triangle--r triangle color-green color-green'></div>
```

Comment:
This is a button. I would replace Line 9 with `<button>` element and include a hover state.